containers/integration-tests-build/Dockerfile

```diff
@@ -17,6 +17,12 @@ RUN wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.k
 RUN wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
 RUN add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod focal main"
 
+# Install gramine for the in-kernel SGX driver
+RUN curl -fsSLo /usr/share/keyrings/gramine-keyring.gpg https://packages.gramineproject.io/gramine-keyring.gpg
+RUN echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/gramine-keyring.gpg] https://packages.gramineproject.io/ focal main' | tee /etc/apt/sources.list.d/gramine.list
+RUN apt update
```

Comment:
Aren't we already doing an apt-get update above?

Reply:
Yup, we need to do it again after adding the gramine gpg key.